src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/mha_single_token.cpp

```diff
@@ -1034,6 +1150,30 @@ void mha_single_token(const ov::intel_cpu::PlainTensor& query,
                                                 past_v_scale_zp,
                                                 head_sum);
         }
+    } else if (query.get_precision() == ov::element::f16) {
+#if defined(__ARM_FEATURE_FP16_VECTOR_ARITHMETIC)
+        if (present_key.get_precision() == ov::element::f16) {
+            mha_single_token_kernel<ov::float16, ov::float16, ov::float16>(query,
```

Comment:
@mory91 I am wondering will mha_single_token_kernel<ov::float16, ov::float16, float> work already now or we need some functions to be implemented? 
I am still have some concerns regarding usage of FP16 as intermediate precision inside SDPA. Would be nice to have fallback option for such cases.